api/database.json

```diff
@@ -203,6 +203,15 @@
         "numberOfShards": 6
       }
     },
+    {
+      "type": "edgecollection",
+      "name": "favourites",
+      "options": {
+        "replicationFactor": 3,
+        "writeConcern": 1,
+        "numberOfShards": 6
+      }
+    },
```

Comment:
I think these changes might have to be added in `/k8s/apps/bases/api/database-configmap.yaml` as well, so the `database-migration` init container will create this collection. We should probably look moving the `database.json` into a good spot where it can be used wherever required.

Reply:
Good catch. Fixed :+1: